spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/LibraryControllerHelperService.java

```diff
@@ -364,6 +364,27 @@ public JSONObject changeLibraryLocation(HttpSession session, JSONObject json) {
 
     return JSONUtils.SimpleJSONResponse("Note saved successfully");
   }
+  
+  public JSONObject changeLibraryIdBarcode(HttpSession session, JSONObject json) {
+    Long libraryId = json.getLong("libraryId");
+    String idBarcode = json.getString("identificationBarcode");
+    
+    try {
+      if (!"".equals(idBarcode)) {
```

Comment:
`org.apache.commons.lang.StringUtils.isEmpty(String)` is available for null safe string checking.